presto-hive/src/test/java/com/facebook/presto/hive/TestOptimizedPartitionedOutputOperatorQueries.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.hive;
+
+import com.google.common.collect.ImmutableMap;
+
+import java.util.Optional;
+
+import static io.airlift.tpch.TpchTable.getTables;
+
+public class TestOptimizedPartitionedOutputOperatorQueries
```

Comment:
What's the reason for having this test in presto-hive? The partitioned output operator is a generic operator and has no dependency on Hive connector. Would it be possible to use TpchQueryRunnerBuilder and move these tests into presto-tests?